sponsors/migrations/0031_auto_20210801_2038.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.24 on 2021-08-01 20:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sponsors', '0030_auto_20210715_2023'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='sponsorbenefit',
+            name='added_by_user',
+            field=models.BooleanField(blank=True, default=False, verbose_name='Added by user?'),
+        ),
+    ]
```

Comment:
Why does this migration exist within this PR? I mean, I don't see changes on `sponsors/models.py` file to generate new migrations.

Reply:
The #1682 includes the [`added_by_user` field with `blank=True`](https://github.com/python/pythondotorg/pull/1682/files#diff-07dd10655f36432bf9245868f1b9b19a10cad6abf5b7807169bc334975275b1cR377), [but the migration](https://github.com/python/pythondotorg/pull/1682/files#diff-82a26abccbb6e7cf641e0fec6f37e9f4edde489ef3f8a0070162ee5176b5bfb2R16) sent with the pull-request does not include `blank=True`.
The django migration framework makes it mandatory to create a migration whenever any changes to the Model, model field, model fields attributes and Meta are made even if this change (`blank=True`) does not generate SQL

Reply:
I investigated a little more, and django 2.0.13 cannot detect the divergence between the `added_by_user` migration and the `added_by_user` model field. As of django 2.1, something has changed in the django migration framework (I couldn't find anything to explain this in the release notes), and the divergence is finally detected, and `python manage.py makemigrations` generates a new migration.

Reply:
the simplest way to reproduce the problem is on the main branch, update as minimum as possible so as not to have errors in the migration framework, and finally run `makemigrations`

```python
pip install "django<2.2,>2.0.13" django-tastypie django-sitetree django-filter==2.0.0 -U
python manage.py makemigrations

```